extensions/cloud-cli-login-statuses/README.md

```diff
@@ -0,0 +1,3 @@
+# Cloud CLI login statuses
+
+Check if you are logged in AWS, Google Cloud and Azure CLIs
```

Comment:
Could you add a bit more context here about how it works and what is required and so on?

Reply:
Done